man/conf.py

```diff
@@ -60,6 +71,10 @@ def _get_manpages():
                 section,
                 )
 
+extensions = [
+    'sphinxarg.ext',
```

Comment:
not sure what we can do, if we want to use an extension which is not packaged by distro when building manpages. please take the hat off: it is our good old friend: missing dependencies in distro.

what i have off the top of my head is to leverage `sphinxarg.ext` as a make-dist-time dependency, and include the generated artifact in our dist tarball.